### PR TITLE
Increase coverage for height, helpers and zadara_helpers

### PR DESCRIPTION
**Pull request by mariavkourouni**

### CODE REVIEW

**Comment by gjn** [comment]
jenkins test

File: chsdi/tests/functional/test_helpers.py [hunk 1]
```diff
@@ -68,6 +69,16 @@ def test_check_url(self):
         url = 'http://admin.ch'
         self.assertEqual(url, check_url(url, config))
 
+    def test_sanitize_url(self):
+        url = 'http://www.cwi.nl:80/%7Eguido/Python.html'
+        result = sanitize_url(url)
+        self.assertEqual(result, result)
```
Comment (gjn) [inline chsdi/tests/functional/test_helpers.py]: This test can never fail. What should be the result?

File: chsdi/tests/functional/test_helpers.py [hunk 2]
```diff
@@ -68,6 +69,16 @@ def test_check_url(self):
...
+
+    def test_sanitize_url_throws_exception(self):
+        # should raise an exception error here (such as IOError, ImportError, ValueError, KeyboardInterrupt or EOFError)
+        url = []
+        self.assertRaises(Exception, sanitize_url(url))
```
Comment (gjn) [inline chsdi/tests/functional/test_helpers.py]: Be as specific as possible. Which Exception is raised when passing in an array?

File: chsdi/tests/functional/test_helpers.py (hunk 1 above)
Comment (gjn) [inline chsdi/tests/functional/test_helpers.py]: Please ignore this comment.

Comment (gjn) [inline chsdi/tests/functional/test_helpers.py]: This test can never fail. (we compare 2 same objects). What's the goal of this test?

File: chsdi/tests/functional/test_helpers.py [hunk 3]
```diff
@@ -77,3 +88,46 @@ def test_transformCoordinate(self):
...
+
+    def test_round(self):
+        testnumber = 4.4
+        result = round(testnumber)
+        self.assertTrue(result == 4, result)
+
+    def test_format_search_text(self):
+        testinput_str = '//'
+        result = format_search_text(testinput_str)
+        self.assertTrue(result, False)
```
Comment (gjn) [inline chsdi/tests/functional/test_helpers.py]: It's assertTrue here, not assertEqual. So the False has no impact and test can never faile.

File: chsdi/tests/functional/test_helpers.py [hunk 4]
```diff
@@ -77,3 +88,46 @@ def test_transformCoordinate(self):
...
+        self.assertTrue(result == 'Hallo', result)
+
+    def test_float_raise_nan(self):
+        testval = 5
+        result = float_raise_nan(testval)
+        self.assertTrue(result, result)
+
+        #testval = 'inf'
+        #result = float_raise_nan(testval)
+        #self.assertTrue(result, result)
```
Comment (gjn) [inline chsdi/tests/functional/test_helpers.py]: Try to add a float that is None and check if an Exception is raised.

**Comment by gjn** [comment]
Very nice first PR!

Added a couple of comments that needs adaptions.

Also, try to squash the commits into one and push force.

File: chsdi/tests/functional/test_helpers.py [hunk 5]
```diff
@@ -77,3 +126,47 @@ def test_transformCoordinate(self):
         self.assertTrue(isinstance(wkt_21781, Geometry))
         self.assertEqual(int(wkt_21781.GetX()), 595324)
         self.assertEqual(int(wkt_21781.GetY()), 84952)
+
+    def test_check_even(self):
+        testnumber = 10
+        result = check_even(testnumber)
+        self.assertEqual(result, True)
```
Comment (loicgasser) [inline chsdi/tests/functional/test_helpers.py]: `self.assertTrue(result)`

File: chsdi/tests/functional/test_helpers.py [hunk 6]
```diff
@@ -77,3 +126,47 @@ def test_transformCoordinate(self):
...
+
+        testnumber = 5
+        result = check_even(testnumber)
+        self.assertEqual(result, False)
```
Comment (loicgasser) [inline chsdi/tests/functional/test_helpers.py]: `self.assertTrue(result)`

File: chsdi/tests/functional/test_helpers.py [hunk 7]
```diff
@@ -77,3 +126,47 @@ def test_transformCoordinate(self):
...
+
+    def test_round(self):
+        testnumber = 4.4
+        result = round(testnumber)
+        self.assertTrue(result == 4, result)
```
Comment (loicgasser) [inline chsdi/tests/functional/test_helpers.py]: `self.assertEqual(result, 4)`

File: chsdi/tests/functional/test_helpers.py [hunk 8]
```diff
@@ -77,3 +126,47 @@ def test_transformCoordinate(self):
...
+        self.assertEqual(result, 'Hallo\\!')
+
+        testinput_str2 = u'über'
+        result2 = format_search_text(testinput_str2)
+        self.assertEqual(result2, 'ueber')
+
+    def test_remove_accents(self):
+        testinput_str = None
+        result = remove_accents(testinput_str)
+        self.assertTrue(result is None, result)
```
Comment (loicgasser) [inline chsdi/tests/functional/test_helpers.py]: àssertEqual

File: chsdi/tests/functional/test_helpers.py [hunk 9]
```diff
@@ -77,3 +126,47 @@ def test_transformCoordinate(self):
...
+
+    def test_escape_sphinx_syntax(self):
+        testinput_str = None
+        result = escape_sphinx_syntax(testinput_str)
+        self.assertTrue(result is None, result)
```
Comment (loicgasser) [inline chsdi/tests/functional/test_helpers.py]: assertEqual

File: chsdi/tests/functional/test_helpers.py [hunk 10]
```diff
@@ -77,3 +126,47 @@ def test_transformCoordinate(self):
...
+
+    def test_quoting(self):
+        testtext = 'Hallo'
+        result = quoting(testtext)
+        self.assertTrue(result == 'Hallo', result)
```
Comment (loicgasser) [inline chsdi/tests/functional/test_helpers.py]: assertEqual

File: chsdi/tests/functional/test_helpers.py [hunk 11]
```diff
@@ -77,3 +126,47 @@ def test_transformCoordinate(self):
...
+
+    def test_float_raise_nan(self):
+        testval = 5
+        result = float_raise_nan(testval)
+        self.assertTrue(result == 5.0, result)
```
Comment (loicgasser) [inline chsdi/tests/functional/test_helpers.py]: assertEqual

File: chsdi/tests/functional/test_helpers.py [hunk 12]
```diff
@@ -77,3 +126,47 @@ def test_transformCoordinate(self):
...
+        self.assertFalse(result == 'ValueError', result)
```
Comment (loicgasser) [inline chsdi/tests/functional/test_helpers.py]: assertRaises

File: chsdi/tests/functional/test_zadara_helpers.py [hunk 13]
```diff
@@ -4,11 +4,36 @@
...
-from chsdi.lib.zadara_helpers import find_files
+from chsdi.lib.zadara_helpers import find_files, hbytes
 
 
 class Test_ZadaraHelpers(unittest.TestCase):
 
+    def test_hbytes(self):
+        testnumber = 5
+        result = hbytes(testnumber)
+        self.assertTrue(result == '5.0 bytes', result)
```
Comment (loicgasser) [inline chsdi/tests/functional/test_zadara_helpers.py]: equal

File: chsdi/tests/functional/test_zadara_helpers.py [hunk 14]
```diff
@@ -4,11 +4,36 @@
...
+
+    def test_hbytes_KB(self):
+        testnumber = 1024.0 + 5
+        result = hbytes(testnumber)
+        self.assertTrue(result == '1.0 KB', result)
```
Comment (loicgasser) [inline chsdi/tests/functional/test_zadara_helpers.py]: equal

File: chsdi/tests/functional/test_zadara_helpers.py [hunk 15]
```diff
@@ -4,11 +4,36 @@
...
+
+    def test_hbytes_MB(self):
+        testnumber = 1024.0 ** 2 + 5
+        result = hbytes(testnumber)
+        self.assertTrue(result == '1.0 MB', result)
```
Comment (loicgasser) [inline chsdi/tests/functional/test_zadara_helpers.py]: equal

File: chsdi/tests/functional/test_zadara_helpers.py [hunk 16]
```diff
@@ -4,11 +4,36 @@
...
+
+    def test_hbytes_GB(self):
+        testnumber = 1024.0 ** 3 + 5
+        result = hbytes(testnumber)
+        self.assertTrue(result == '1.0 GB', result)
```
Comment (loicgasser) [inline chsdi/tests/functional/test_zadara_helpers.py]: equal

File: chsdi/tests/functional/test_zadara_helpers.py [hunk 17]
```diff
@@ -4,11 +4,36 @@
...
+
+    def test_hbytes_TB(self):
+        testnumber = 1024.0 ** 4 + 5
+        result = hbytes(testnumber)
+        self.assertTrue(result == '1.0 TB', result)
```
Comment (loicgasser) [inline chsdi/tests/functional/test_zadara_helpers.py]: equal

**Comment by loicgasser** [comment]
ok great!

**Pull request merged**